services/venice-router/src/main/java/com/linkedin/venice/router/stats/SecurityStats.java

```diff
@@ -49,4 +51,16 @@ public void recordNonSslRequest() {
   private void recordLiveConnectionCount() {
     this.sslLiveConnectionCount.record(secureConnectionCountSupplier.getAsInt());
   }
+
+  public void registerSslHandshakeSensors(SslInitializer sslInitializer) {
+    registerSensor(
+        new AsyncGauge(
+            (ignored1, ignored2) -> sslInitializer.getHandshakesStarted()
+                - (sslInitializer.getHandshakesSuccessful() + sslInitializer.getHandshakesFailed()),
```

Comment:
Why are we not using `SslInitializer#getPendingHandshakes()`? Is it not reliable?

Reply:
Right, according to Antony, this call is expensive.